pennylane/qchem/hamiltonian.py

```diff
@@ -170,21 +175,30 @@ def _fermionic_hamiltonian(*args):
             *args (array[array[float]]): initial values of the differentiable parameters
 
         Returns:
-            tuple(array[float], list[list[int]]): the Hamiltonian coefficients and operators
+            Union[FermiSentence, tuple(array[float], list[list[int]])]: fermionic Hamiltonian
         """
+
         core_constant, one, two = electron_integrals(mol, core, active)(*args)
 
-        return fermionic_observable(core_constant, one, two, cutoff)
+        if not fs:
+            warnings.warn(
+                "This function will return a fermionic operator by default in the next release. For"
+                " details, see the Fermionic Operators tutorial:"
+                " https://pennylane.ai/qml/demos/tutorial_fermionic_operators"
```

Comment:
> Should we add something like: "to return a fermionic operator already, use the fs kwarg." ?

Added,

Also will the fs kwarg stick around, or is it going to be deleted once qml.qchem.jordan_wigner is deleted?

It will be removed after the upcoming release.